wandb/integration/tensorboard/log.py

```diff
@@ -15,9 +15,10 @@
 tensor_util = wandb.util.get_module("tensorboard.util.tensor_util")
 
 
+# prefer tensorboard, fallback to protobuf in tensorflow when tboard isn't available
 pb = wandb.util.get_module(
     "tensorboard.compat.proto.summary_pb2"
-) or wandb.util.get_module("tensorflow.summary")
+) or wandb.util.get_module("tensorflow.core.framework.summary_pb2")
```

Comment:
We generally weren't getting here because `sync_tensorboard` currently requires the tensorboard package, but all recent versions of tensorflow do not have `Summary` available at `tensorflow.summary` and I verified all old and new versions have the `summary_pb2` module.